Update control

smart-sicher opened the pull request:
default-jre is outdated

smart-sicher committed:
Update control

default-jre is outdated

anticapitalista commented:
Thank you but we no longer use github, but GitLab



On 2024-11-24 22:29, Aika wrote:
> default-jre is outdated 
> -------------------------
> 
> YOU CAN VIEW, COMMENT ON, OR MERGE THIS PULL REQUEST ONLINE AT:
> 
>   https://github.com/antiX-Linux/1-to-1-assistance-antix/pull/1 
> 
> COMMIT SUMMARY
> 
>  	* 5352c6f [1] Update control
> 
> FILE CHANGES 
> 
> (1 file [2]) 
> 
>  	* M debian/control [3] (2) 
> 
> PATCH LINKS:
> 
>  	*
> https://github.com/antiX-Linux/1-to-1-assistance-antix/pull/1.patch
>  	* https://github.com/antiX-Linux/1-to-1-assistance-antix/pull/1.diff
> 
> --
> Reply to this email directly, view it on GitHub [4], or unsubscribe
> [5].
> You are receiving this because you are subscribed to this
> thread.Message ID:
> ***@***.***>
> 
> Links:
> ------
> [1] https://github.com/antiX-Linux/1-to-1-assistance-antix/pull/1/commits/5352c6fe40b9c59dba11e6b429ed336aedcbf7f4
> [2] https://github.com/antiX-Linux/1-to-1-assistance-antix/pull/1/files
> [3] https://github.com/antiX-Linux/1-to-1-assistance-antix/pull/1/files#diff-2a13f1344504379e877324d3a0375adcbcb4cb27d7e575ad8f4618a52d6a00e0
> [4] https://github.com/antiX-Linux/1-to-1-assistance-antix/pull/1
> [5] https://github.com/notifications/unsubscribe-auth/AADGKFWMEIXGPGAP4SZZU432CIZJBAVCNFSM6AAAAABSMTARZ6VHI2DSMVQWIX3LMV43ASLTON2WKOZSGY4DQMZRGIYDSNY

anticapitalista commented:
Here is the new git repo we use

https://gitlab.com/antiX-Linux/1-to-1-assistance-antix

On 2024-11-25 14:59, ***@***.*** wrote:
> Thank you but we no longer use github, but GitLab
> 
> 
> 
> On 2024-11-24 22:29, Aika wrote:
>> default-jre is outdated 
>> -------------------------
>> 
>> YOU CAN VIEW, COMMENT ON, OR MERGE THIS PULL REQUEST ONLINE AT:
>> 
>>   https://github.com/antiX-Linux/1-to-1-assistance-antix/pull/1 
>> 
>> COMMIT SUMMARY
>> 
>>  	* 5352c6f [1] Update control
>> 
>> FILE CHANGES 
>> 
>> (1 file [2]) 
>> 
>>  	* M debian/control [3] (2) 
>> 
>> PATCH LINKS:
>> 
>>  	*
>> https://github.com/antiX-Linux/1-to-1-assistance-antix/pull/1.patch
>>  	* https://github.com/antiX-Linux/1-to-1-assistance-antix/pull/1.diff
>> 
>> --
>> Reply to this email directly, view it on GitHub [4], or unsubscribe
>> [5].
>> You are receiving this because you are subscribed to this
>> thread.Message ID:
>> ***@***.***>
>> 
>> Links:
>> ------
>> [1] https://github.com/antiX-Linux/1-to-1-assistance-antix/pull/1/commits/5352c6fe40b9c59dba11e6b429ed336aedcbf7f4
>> [2] https://github.com/antiX-Linux/1-to-1-assistance-antix/pull/1/files
>> [3] https://github.com/antiX-Linux/1-to-1-assistance-antix/pull/1/files#diff-2a13f1344504379e877324d3a0375adcbcb4cb27d7e575ad8f4618a52d6a00e0
>> [4] https://github.com/antiX-Linux/1-to-1-assistance-antix/pull/1
>> [5] https://github.com/notifications/unsubscribe-auth/AADGKFWMEIXGPGAP4SZZU432CIZJBAVCNFSM6AAAAABSMTARZ6VHI2DSMVQWIX3LMV43ASLTON2WKOZSGY4DQMZRGIYDSNY

smart-sicher committed:
Update compat

Compatibility levels before 10 are deprecated

smart-sicher committed:
Update 1-to-1_assistance.sh

no default-jre requires

smart-sicher committed:
Update changelog

new changes

smart-sicher committed:
Update 1-to-1_assistance_receive.sh

changed for vinagre and x11vnc password

smart-sicher committed:
Update control

changed for vinagre

smart-sicher committed:
Update changelog

changed for vinagre and x11vnc password

smart-sicher committed:
Update 1-to-1_assistance_receive.sh

Add VNC_PASSWORD

smart-sicher committed:
Update 1-to-1_assistance_receive.sh

Add VNC_PASSWORD

smart-sicher committed:
Update 1-to-1_assistance_receive.sh

Add VNC_PASSWORD

smart-sicher committed:
Update changelog

Add VNC_PASSWORD query

smart-sicher committed:
Update 1-to-1_assistance_provide.sh

vinagre but no ssvnc

smart-sicher committed:
Update 1-to-1_assistance_provide.sh

Change command for vinagre

smart-sicher committed:
Update changelog

Added VNC_PASSWORD query and changed command for Vinagre

smart-sicher committed:
Update changelog

Changed for use vinagre and x11vnc password

smart-sicher committed:
Update 1-to-1_assistance.sh

replaces ssvnc with vinagre

smart-sicher committed:
Update 1-to-1_assistance_receive.sh

Request a password for x11vnc server - comment changed

smart-sicher committed:
Update install

Add 1-to-1_assistance.desktop

smart-sicher committed:
Update not_preinst

no install ssvnc

smart-sicher committed:
Delete supplementary/ssvnc.desktop

no install ssvnc

smart-sicher committed:
Delete supplementary/x11vnc.desktop

x11vnc is installed as a dependency

smart-sicher committed:
Add files via upload

Starter file was in the wrong folder

smart-sicher committed:
Update 1-to-1_assistance.desktop

1-zu-1 Reverse-VNC, reverse and simple

smart-sicher committed:
Update 1-to-1_assistance.desktop

1-zu-1 Reverse-VNC, reverse and simple

smart-sicher committed:
Delete skel/.ssvncrc

vinagre replaces ssvnc

smart-sicher committed:
Add files via upload

Adaptation to Vinagre

smart-sicher committed:
Delete skel/.1-to-1_assistance_receive_rc

Adaptation to Vinagre

smart-sicher committed:
Add files via upload

Adaptation to Vinagre

smart-sicher committed:
Update install

Adaptation to Vinagre

smart-sicher committed:
Delete debian/changelog

Adaptation to Vinagre

smart-sicher committed:
Add files via upload

Adaptation to Vinagre

smart-sicher committed:
Update changelog

Adaptation to Vinagre

smart-sicher committed:
Delete bin/1-to-1_assistance_receive.sh

Adaptation to Vinagre

smart-sicher committed:
Delete bin/1-to-1_assistance_provide.sh

Adaptation to Vinagre

smart-sicher committed:
Add files via upload

Adaptation to Vinagre

smart-sicher committed:
Add files via upload

Adaptation to Vinagre

smart-sicher committed:
Update 1-to-1_assistance_receive.sh

Adaptation to Vinagre

smart-sicher committed:
Update 1-to-1_assistance_provide.sh

Adaptation to Vinagre

smart-sicher committed:
Add files via upload

Starter for help site 1-to-1_assistance.html

smart-sicher committed:
Delete supplementary/faq_1-to-1_assistance.desktop

Menu warning in Xfce

smart-sicher committed:
Delete 1-to-1_assistance directory

now online =>  http://aika.bplaced.net/1-to-1_vnc-help/

smart-sicher committed:
Delete bin/1-to-1_assistance.sh

delete for replace

smart-sicher committed:
Add files via upload

add online help => http://aika.bplaced.net/1-to-1_vnc-help/

smart-sicher committed:
Delete 1-to-1_assistance.html

help file now online

smart-sicher committed:
Delete antiX-boot.conf

no need in Debian Linux

smart-sicher committed:
Delete debian/install

delete for replace

smart-sicher committed:
Add files via upload

remove image-folder and remove html-file

smart-sicher committed:
Delete debian/control

delete for replace

smart-sicher committed:
Add files via upload

Depends: yad, vinagre, x11vnc, stunnel4, bash, grep, lxrandr, x11-xserver-utils

smart-sicher committed:
Delete bin/1-to-1_assistance.sh

delete for replace

smart-sicher committed:
Add files via upload

Depends: yad, vinagre, x11vnc, stunnel4, bash, grep, lxrandr, x11-xserver-utils

smart-sicher committed:
Delete debian/control

delete for replace

smart-sicher committed:
Add files via upload

Depends: yad, vinagre, x11vnc, stunnel4, bash, grep, lxrandr, x11-xserver-utils

smart-sicher committed:
Delete debian/install

delete for replace

smart-sicher committed:
Add files via upload

HTML file and image folder removed,
1-to-1_vnc-help is online

smart-sicher committed:
Delete bin/1-to-1_assistance.sh

delete for replace

smart-sicher committed:
Add files via upload

Depends: yad, vinagre, x11vnc, stunnel4, bash, grep, lxrandr, x11-xserver-utils

smart-sicher committed:
Delete bin/1-to-1_assistance_provide.sh

delete for new upload

smart-sicher committed:
Add files via upload

smart-sicher committed:
Update control

smart-sicher committed:
Update changelog

smart-sicher committed:
Delete bin/1-to-1_assistance_provide.sh

delete for new upload

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete bin/1-to-1_assistance_provide.sh

delete for new upload

smart-sicher committed:
Add files via upload

smart-sicher committed:
Update 1-to-1_assistance_provide.sh

smart-sicher committed:
Update control

vinagre, x11vnc, stunnel4, bash, grep, arandr, yad, coreutils, x11-xserver-utils

smart-sicher committed:
Update 1-to-1_assistance.sh

vinagre, x11vnc, stunnel4, bash, grep, arandr, yad, coreutils, x11-xserver-utils

smart-sicher committed:
Delete bin/1-to-1_assistance_receive.sh

delete for replace

smart-sicher committed:
Add files via upload

arandr bug corrected

smart-sicher committed:
Delete debian/changelog

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete bin/1-to-1_assistance.sh

delete for replace

smart-sicher committed:
Delete bin/1-to-1_assistance_provide.sh

delete for replace

smart-sicher committed:
Delete bin/1-to-1_assistance_receive.sh

delete for replace

smart-sicher committed:
Create 1-to-1_assistance.sh

smart-sicher committed:
Update 1-to-1_assistance.sh

version 1.3

smart-sicher committed:
Add files via upload

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete bin/1-to-1_assistance_provide.sh

smart-sicher committed:
Delete bin/1-to-1_assistance_receive.sh

smart-sicher committed:
Add files via upload

smart-sicher committed:
Add files via upload

smart-sicher committed:
Update 1-to-1_assistance_provide.sh

version 1.3

smart-sicher committed:
Update 1-to-1_assistance_receive.sh

version 1.3

smart-sicher committed:
Delete bin/1-to-1_assistance_provide.sh

delete for replace

smart-sicher committed:
Delete bin/1-to-1_assistance_receive.sh

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete bin/1-to-1_assistance.sh

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete bin/1-to-1_assistance_receive.sh

delete for replace

smart-sicher committed:
Delete bin/1-to-1_assistance.sh

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete bin/1-to-1_assistance_provide.sh

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete bin/1-to-1_assistance.sh

delete for replace

smart-sicher committed:
Delete bin/1-to-1_assistance_receive.sh

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete bin/1-to-1_assistance_provide.sh

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete bin/1-to-1_assistance_receive.sh

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete debian/control

delete for replace

smart-sicher committed:
Add files via upload

vinagre, x11vnc, stunnel4, bash, grep, yad, coreutils, libgtk-3-0, libgtk2.0-0

smart-sicher committed:
Delete bin/1-to-1_assistance_receive.sh

delete for replace

smart-sicher committed:
Delete bin/1-to-1_assistance_provide.sh

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete bin/1-to-1_assistance.sh

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete bin/1-to-1_assistance_receive.sh

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Update 1-to-1_assistance_receive.sh

smart-sicher committed:
Delete bin/1-to-1_assistance_receive.sh

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete bin/1-to-1_assistance_receive.sh

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete bin/1-to-1_assistance.sh

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete debian/install

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete debian/control

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete bin/1-to-1_assistance_receive.sh

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete skel/.1-to-1_assistance_receive_rc

delete for replace

smart-sicher committed:
Add files via upload

smart-sicher committed:
Delete skel/.1-to-1_assistance_receive_rc_

delete for replace

smart-sicher committed:
Add files via upload
